src/http/request.cr

```diff
@@ -16,6 +16,8 @@ class HTTP::Request
     end
   end
 
+  # Returns a convenience wrapper around querying and setting cookie related
+  # headers, see *HTTP::Cookies*
```

Comment:
Ah, not too intuitive tbh, since it's also used for inline code. I prefer YARD's `{Foo}`.